src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/Header/Header.tsx

```diff
@@ -0,0 +1,68 @@
+import { useFormContext } from 'react-hook-form';
+import { Slider } from '@mui/material';
+
+import { Svg } from 'shared/components';
+import {
+  theme,
+  StyledClearedButton,
+  StyledFlexColumn,
+  StyledFlexTopCenter,
+  StyledLabelBoldLarge,
+  StyledLabelLarge,
+} from 'shared/styles';
+
+import { HeaderProps } from './Header.types';
+import { StyledImg, StyledSliderPanelHeader } from './Header.styles';
+
+const commonButtonStyles = {
+  p: theme.spacing(1),
+  mr: theme.spacing(0.2),
+};
+
+export const Header = ({
+  name,
+  label,
+  isExpanded,
+  isMultiple,
+  onArrowClick,
+  onTrashClick,
+}: HeaderProps) => {
+  const { getValues } = useFormContext();
+
+  const { min, max, minLabel, maxLabel, minImage, maxImage } = getValues(name);
```

Comment:
is the data updated if some value changes?
maybe it's better to use watch?

Reply:
done